component/src/main/resources/installation-scripts/uninstall.bat

```diff
@@ -27,37 +37,37 @@ SET module_name=mongodb
 
 IF NOT EXIST "%ballerina_lib_location%\wso2-%module_name%-module-%version%.jar" (
     IF NOT EXIST "%ballerina_balo_location%\wso2\%module_name%\0.0.0\%module_name%.zip" (
-	   ECHO MongoDB module is not installed!
+	   ECHO [WARNING] MongoDB module is not installed!
 	   GOTO :END
 	)
 )
 
 IF EXIST "%ballerina_lib_location%\wso2-%module_name%-module-%version%.jar" (
    DEL "%ballerina_lib_location%\wso2-%module_name%-module-%version%.jar"
    IF EXIST "%ballerina_lib_location%\wso2-%module_name%-module-%version%.jar" (
-    ECHO An error occurred while deleting %ballerina_lib_location%wso2-%module_name%-module-%version%.jar
+    ECHO [WARNING] An error occurred while deleting %ballerina_lib_location%wso2-%module_name%-module-%version%.jar
 	GOTO :FAILED_JAR_DELETION
    )
 )
 
 IF EXIST "%ballerina_balo_location%\wso2\%module_name%\0.0.0\%module_name%.zip" (
    DEL "%ballerina_balo_location%\wso2\%module_name%\0.0.0\%module_name%.zip"
    IF EXIST "%ballerina_balo_location%\wso2\%module_name%\0.0.0\%module_name%.zip" (
-    ECHO An error occurred while deleting %ballerina_balo_location%wso2\%module_name%\0.0.0\%module_name%.zip
+    ECHO [WARNING] An error occurred while deleting %ballerina_balo_location%wso2\%module_name%\0.0.0\%module_name%.zip
 	GOTO :FAILED_BALO_DELETION
    )
 )
 
 :SUCCESS
-ECHO Successfully uninstalled MongoDB module!
+ECHO [INFO] Successfully uninstalled MongoDB module: wso2-%module_name%-module-%version%
 GOTO :END
 
 :FAILED_JAR_DELETION
-ECHO Un-installation is incomplete due to an error. Please manually delete %ballerina_lib_location%wso2-%module_name%-module-%version%.jar and %ballerina_balo_location%wso2\%module_name%\0.0.0\%module_name%.zip
+ECHO [WARNING] Un-installation is incomplete due to an error. Please manually delete %ballerina_lib_location%wso2-%module_name%-module-%version%.jar and %ballerina_balo_location%wso2\%module_name%\0.0.0\%module_name%.zip
```

Comment:
This is an `[ERROR]` rather a `[WARNING]` right ?
Same goes with `:FAILED_BALO_DELETION` too.